main.nf

```diff
@@ -19,4 +30,39 @@ workflow mtbseq {
 
 }
 
+workflow per_sample {
+    reads_ch = Channel.fromFilePairs(params.reads)
+
+    TBBWA(reads_ch, params.gatk38_jar, params.user)
+    TBREFINE(TBBWA.out.next_step, params.gatk38_jar, params.user)
+    TBPILE(TBREFINE.out.next_step, params.gatk38_jar, params.user)
+    TBLIST(TBPILE.out.next_step, params.gatk38_jar, params.user)
+    TBVARIANTS(TBLIST.out.next_step, params.gatk38_jar, params.user)
+    TBSTATS(TBBWA.out.next_step,TBLIST.out.next_step, params.gatk38_jar, params.user)
+    TBSTRAINS(TBLIST.out.next_step, params.gatk38_jar, params.user)
+
+
+
+}
 
+workflow cohort {
```

Comment:
I'm thinking that for the main workflow file i.e. `main.nf` it should be as simple as possible and should not contain these `subworkflows` as top level. 

The `COHORT_ANALYSIS` and `PER_SAMPLE_ANALYSIS` should be in the `workflows/WORKFLOW_NAME/WORKFLOW_FILE` folder.

**NOTE:** Also the names of the workflows (apart from `test`) should be in all capitals.

Reply:
ok! I'll add these workflows on subfolders and include them on the main folder